packages/pointer_interceptor/pointer_interceptor_ios/example/ios/RunnerTests/RunnerTests.swift

```diff
@@ -0,0 +1,26 @@
+import Flutter
+import UIKit
+import XCTest
+
+@testable import pointer_interceptor_ios
+
+// This demonstrates a simple unit test of the Swift portion of this plugin's implementation.
+//
+// See https://developer.apple.com/documentation/xctest for more information about using XCTest.
+
+class RunnerTests: XCTestCase {
+
+  func testGetPlatformVersion() {
+    let plugin = PointerInterceptorIosPlugin()
+
+    let call = FlutterMethodCall(methodName: "getPlatformVersion", arguments: [])
```

Comment:
We should replace this with some test of the actual plugin. E.g., that it returns a clear widget in non-debug and a non-clear widget in debug.